website/content/v1.4/introduction/quickstart.md

```diff
@@ -14,21 +14,10 @@ The easiest way to try Talos is by using the CLI (`talosctl`) to create a cluste
 
 Download `talosctl`:
 
-##### `amd64`
-
 ```bash
-curl -Lo /usr/local/bin/talosctl https://github.com/siderolabs/talos/releases/download/{{< release >}}/talosctl-$(uname -s | tr "[:upper:]" "[:lower:]")-amd64
-chmod +x /usr/local/bin/talosctl
+curl --proto '=https' --tlsv1.2 -sSfL https://talos.dev/install | sh
```

Comment:
I have mixed feeling about using the script.

The first problem is that it always downloads the latest `talosctl`, while it should probably download the latest patch release for the branch I'm using. Otherwise there might be surprises if the version of Talos and talosctl are too far apart.

The second problem is that it simply doesn't work out of the box, it should either try with `sudo`, or try a different location to install (?)

```
$ curl --proto '=https' --tlsv1.2 -sSfL https://talos.dev/install | sh 
Downloading talosctl-linux-amd64...
  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
                                 Dload  Upload   Total   Spent    Left  Speed
  0     0    0     0    0     0      0      0 --:--:-- --:--:-- --:--:--     0
  0     0    0     0    0     0      0      0 --:--:-- --:--:-- --:--:--     0
100 69.8M  100 69.8M    0     0  4891k      0  0:00:14  0:00:14 --:--:-- 5376k
Download complete!

Validating checksum...
Checksum valid.

mv: cannot move '/tmp/talosctl.TgEZVv/talosctl-linux-amd64' to '/usr/local/bin/talosctl': Permission denied
```

Reply:
2nd part of your comment is fixed now.

As for the 1st part, the only time this script is used/documented is in the "Getting Started" guides, so I think it is fair to say the users won't have a release yet. Even so, if the script finds a copy of `talosctl` at the destination, it won't overwrite it so users will keep their current version without breaking anything.